### ISSUE TITLE
Wording in Episode 03

### ISSUE THREAD

**Issue by crlna16** [carpentries-incubator/deep-learning-intro#249]
In Episode 03, it should read:

Not everybody agrees on the terminology of **test set versus validation set**. You might find examples in literature where these terms are used the other way around.

Instead of

Not everybody agrees on the terminology of training set versus validation set. You might find examples in literature where these terms are used the other way around.

**Comment by dafnevk** [comment]
Actually, we meant to refer to different uses of 'test set' and 'validation set' . I think most literature agrees on the meaning of 'training set', but the held-out dataset to use for optimizing hyperparameters is sometimes called 'validation set' (which is what we use as well) and sometimes 'test set'.
Apparently our intention was not clear to you from the current wording. Would you have a suggestion to reword this so that it becomes more clear?

**Comment by crlna16** [comment]
Yes that is what I meant, it is written "training set versus validation set", but I think it should read "test set versus validation set". Sorry if that was unclear in the issue above

**Comment by dafnevk** [comment]
I'm sorry, I did not read your message well, that's indeed exactly what you wrote. Yes, this should be reworded. You're welcome to create a PR if you'd like.

**Comment by dsmits** [comment]
Related: #253

**Comment by cpranav93** [comment]
PR #268 addressed this.
